packages/venia-concept/src/components/CreateAccount/asyncValidators.js

```diff
@@ -0,0 +1,22 @@
+import { RestApi } from '@magento/peregrine';
+
+const { request } = RestApi.Magento2;
+
+export const validateEmail = async value => {
+    try {
+        const body = {
+            customerEmail: value,
+            website_id: null
+        };
+
+        // response is a boolean
+        const available = await request('/rest/V1/customers/isEmailAvailable', {
+            method: 'POST',
+            body: JSON.stringify(body)
+        });
+
+        return !available ? 'This email address is not available.' : null;
+    } catch (error) {
+        throw 'An error occurred while looking up this email address.';
+    }
+};
```

Comment:
The next version of `informed` that we're moving to in #1078 does not support `asyncValidators`. We have decided to keep this code around in the hopes that support for it gets added back. Then we can just hook up this code to the UI.